spec/helpers/articles_helper_spec.rb

```diff
@@ -1,25 +1,29 @@
-RSpec.describe ArticlesHelper, '#category_has_more_content?', type: :helper do
-  let(:article)                  { double(categories: [category]) }
-  let(:category_id)              { 'category-id' }
-  let(:category)                 { Core::Category.new(category_id, contents: contents) }
-  let(:contents)                 { Array.new(4) { double } }
-  let(:related_content_category) { Core::Category.new(category_id, contents: contents) }
+RSpec.describe ArticlesHelper, type: :helper do
```

Comment:
I know I made this change — but is it needed in this PR since we didn't end up modifying the helper?